api/repositories.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, mongo_collection, repository_collection):
     def add_room(self, house_id, name):
         house_rooms = self.get_rooms_for_house(house_id)
         for room in house_rooms:
-            other_name = room.get_room_attributes()['name']
+            other_name = room.get_room_attributes().name
```

Comment:
You're now mixing two approaches. 
You should go for room.name (room.get_room_attributes()['name'] would also work but is less efficient and uglier).